espresso-server/app/src/test/java/io/appium/espressoserver/test/helpers/InvocationOperationTest.java

```diff
@@ -0,0 +1,80 @@
+package io.appium.espressoserver.test.helpers;
+
+import android.support.annotation.NonNull;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.Executor;
+
+import io.appium.espressoserver.lib.helpers.InvocationOperation;
+
+import static junit.framework.Assert.fail;
+import static org.junit.Assert.*;
+
+public class InvocationOperationTest {
```

Comment:
It would be also good to have one unit test where arguments are used and one test where more than one method is called in a chain.